tests/Aspire.Dashboard.Components.Tests/Shared/TestThemeResolver.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Model;
+
+namespace Aspire.Dashboard.Components.Tests.Shared;
+
+public sealed class TestThemeResolver : IThemeResolver
+{
+    public string EffectiveTheme { get; set; } = "Dark";
```

Comment:
The rest of the code uses lowercase values here. I know this is test code, but better to be consistent.

```suggestion
    public string EffectiveTheme { get; set; } = "dark";
```